modules/nodes-pods-vertical-autoscaler-custom-resource.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-vertical-autoscaler.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nodes-pods-vertical-autoscaler-custom-resource_{context}"]
+= Using the Vertical Pod Autoscaler Operator with Custom Resources
+
+The Vertical Pod Autoscaler Operator (VPA) can update not only built-in resources such as deployments or stateful sets, but also custom resources that manage pods.
+
+In order to use the VPA with a custom resource, when you create the the `CustomResourceDefinition` (CRD) object, you must configure the `labelSelectorPath` field in the `/scale` subresource. The `/scale` subresource creates a `Scale` object. The `labelSelectorPath` field defines the JSON path inside the custom resource that corresponds to `Status.Selector` in the `Scale` object and in the custom resource. The following is an example of a `CustomResourceDefinition` and a `CustomResource` that fulfills these requirements, along with a `VerticalPodAutoscaler` definition that targets the custom resource. The following example shows the `/scale` subresource contract. 
+
+[NOTE]
+====
+Using this example will not result in the VPA scaling pods because there is no controller for the custom resource that allows it to own any pods. As such, you must write a controller in a language supported by Kubernetes to manage the reconciliation and state management between the custom resource and your pods. The example is intended to illustrate the configuration for the VPA to understand the custom resource as scalable.
```

Comment:
Nit: ISG recommends avoiding future tense: https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense

Though if you feel that switching to present tense would be awkward, that is your call.